Patches/SQLite3/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+project ( SQLite3 )
+cmake_minimum_required(VERSION 2.8)
+
+# Build output paths
+if( NOT CMAKE_ARCHIVE_OUTPUT_DIRECTORY )
+  set( CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${SQLite3_BINARY_DIR}/lib )
+endif()
+if( NOT CMAKE_LIBRARY_OUTPUT_DIRECTORY )
+  set( CMAKE_LIBRARY_OUTPUT_DIRECTORY ${SQLite3_BINARY_DIR}/lib )
+endif()
+if( NOT CMAKE_RUNTIME_OUTPUT_DIRECTORY )
+  set( CMAKE_RUNTIME_OUTPUT_DIRECTORY ${SQLite3_BINARY_DIR}/bin )
+endif()
+
+# Default to building shared
+option(BUILD_SHARED_LIBS "Build with shared libraries." OFF)
+
+# We need -fPIC when building statically
+if( UNIX AND NOT BUILD_SHARED_LIBS)
+  set (CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+endif()
```

Comment:
Is there any reason shared should not also build PIC? Also, as I believe this does nothing on Windows, I believe the conditional is unneeded.